crates/transaction-pool/src/traits.rs

```diff
@@ -348,6 +352,12 @@ impl PoolTransaction for PooledTransaction {
         }
     }
 
+    /// Returns the transaction's [`TransactionKind`], which is the address of the recipient or
+    /// [`TransactionKind::Create`] if the transaction is a contract creation.
+    fn kinds(&self) -> &TransactionKind {
```

Comment:
```suggestion
    fn kind(&self) -> &TransactionKind {
```

Reply:
Thanks for comment. I follow up the change request w/ edfaa6c12bfb78882a88c3cde97b14dcad7c70a5.